src/main/java/io/debezium/connector/yugabytedb/connection/ReplicationConnection.java

```diff
@@ -103,7 +103,7 @@ interface Builder {
          */
         String DEFAULT_SLOT_NAME = "debezium";
         String DEFAULT_PUBLICATION_NAME = "dbz_publication";
-        boolean DEFAULT_DROP_SLOT_ON_CLOSE = true;
+        boolean DEFAULT_DROP_SLOT_ON_CLOSE = false;
```

Comment:
Is there a reason to change the default here?

Reply:
The postgres connector has the default value of `DROP_SLOT_ON_CLOSE` as false, hence I changed it to false here.

Reply:
Checked in the upstream connector code, this field is set to true. While declaring the config property `drop.slot.on.stop` they are setting its default to false. Will make similar changes here.